works and about section

tomasesquivelgc opened the pull request:
## This branch adds:

### Works section
  - "yoga girl" project
  - 6 other placeholder projects (as in template)

### About section
  - about me section
  - knowledge section

everything as in template

tomasesquivelgc committed:
create work-section

 - add display: grid
 - add grid1 title
 - add grid2 "yoga-image" (first work)

tomasesquivelgc committed:
add text

add text to grid2 (h2 and p)
changed sizes of yoga-img.jpg

tomasesquivelgc committed:
grid2 (yoga) finish

add list of languages
add "see project" button

tomasesquivelgc committed:
grid3 add

add title, p, and background image

tomasesquivelgc committed:
work section finished

grid3 add language li and  see project button
copied grid3 5 times (total of 6 grid3)

tomasesquivelgc committed:
create about-me section

add h1 "about me"
add p element
add button
add background image (not positioned correctly yet)

tomasesquivelgc committed:
finish about-me div

add background image vector

tomasesquivelgc committed:
add knowledge

second part of about section
add ul language and li h3 language (backround image for h3 element will be removed for better alignment)

tomasesquivelgc committed:
finished knoledge subsection

instert image into h3 for better alignment
add more listings to fill quota
background image for about section place correctly

tomasesquivelgc committed:
ul fix

knowledge changed into one big ul containing every other li

tomasesquivelgc committed:
order change in css accord to linters

Kamba56 commented on index.html:
- [x] Kindly use an indentation of two spaces in all your documents. This is a convention and also best practice.

Kamba56 commented on index.html:
```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                <button class="see-project1">See project</button>
+            </div>
```
- [x] You are expected to stick to design template as much as possible as demanded by project requirements. This includes font sizes, colors, margins, paddings, font families and backgrounds. Kindly consider the following
  - Add some spaces between the list of technologies. It gets too tight at 375px screen size. Also, reduce the sizes a bit to match the figma template and so it can fit into the screen.
  - Kindly use the right font for your texts. The font used is `Inter` for the descriptions and `Crete Round` for the headings you can get it from google fonts.
  - The description and list of technologies looks like it is overflowing to the left, kindly adjust it please so it sits at the center of the page.
### Your design
![image](https://user-images.githubusercontent.com/94551484/227733337-155f3729-d238-4651-a88c-dc930fecb0ca.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/227733376-83595f20-0c1f-4dd7-b8fb-aa9f4437234b.png)

Kamba56 commented on index.html:
```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
```
- [x] Kindly reduce the font sizes of the heading and the descriptions a bit to match the template. Also, reduce the spaces to the left and right of the card so that the card contents can fit into it.
- [x] Kindly reduce the sizes of the technologies list, and remove the border so that it matches the design template. 
### Your design
![image](https://user-images.githubusercontent.com/94551484/227733651-dd0deb3f-b42a-45dd-a498-ddd241835e35.png)
### FIgma
![image](https://user-images.githubusercontent.com/94551484/227733666-6c528456-65ad-48ee-9cca-250f83f68e14.png)

Kamba56 commented on index.html:
```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
```
- [x] Kindly add a pressed and hover state to all your buttons to match the design template. You can use the `:hover` and `:active` pseudoclasses for this.
![image](https://user-images.githubusercontent.com/94551484/227733749-0c4f46b5-8573-442e-8a68-baa14703c05a.png)

Kamba56 commented on index.html:
```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+        </div>
+    </section>
+    <section class="about-section">
+        <div class="about-me">
+            <h1>About me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <button class="get-resume">Get my resume</button>
+        </div>
```
- [x] Kindly make sure the `about me` heading sits in one line. You are expected to stick to design as much as possible.
- [x] Kindly reduce the size of the button to match the figma template.
- [x] Kindly bring the background image a bit downwards to match the given template
- [x] Please use the right font family for the text. You are expected to use the same font as the design.
### Your design
![image](https://user-images.githubusercontent.com/94551484/227733997-558609ec-64c3-4990-b79d-f3502a5ec123.png)
### FIgma
![image](https://user-images.githubusercontent.com/94551484/227734031-0e3cfc6d-4077-4488-b456-43a415d92173.png)

Kamba56 commented on index.html:
```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <button class="get-resume">Get my resume</button>
+        </div>
+        <ul class="knowledge">
+                <li><h2><img src="./Normal_Button/language-rectangle.png"> Language</h2></li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li><h2><img src="Normal_Button/frameworks-rectangle.png">   Frameworks</h2></li>
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li><h2><img src="Normal_Button/skills-circle.png"> Skills</h2></li>
+                <li>Database Managment</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Development</li>
+        </ul>
+    </section>
```
- [x] Kindly consider using a nested unordered list to markup your list of technologies. Note that 'languages', 'frameworks', and 'skills' are list items of the parent unordered list and 'html', 'css', and 'javascirpt' are list items under languages. THis is better for HTML semantics. Kindly use the markup below as a guide;
```html

<ul>
  <li>Languages
    <ul>
        <li>JavaScript</li>
        <li>HTML</li>
        <li>CSS</li>
    </ul>
  </li>
  <li>Frameworks
    <ul>
        <li>React</li>
        <li>Ruby on Rails</li>
        <li>Rspec</li>
    </ul>
  </li>
  <li>Skills
    <ul>
        <li>Version control</li>
    ...
    </ul>
  </li>
</ul>
```

Kamba56 commented on index.html:
```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+    </section>
+    <section class="about-section">
+        <div class="about-me">
+            <h1>About me</h1>
```
- [x] Please use only one h1 tag in each of your document. This is best SEO practice. Kindly change this h1 tag to any other heading tag between h2 and h5

Kamba56 commented on index.html:
```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
     </section>
+    <section class="work-section" id="work-section">
+        <div class="work">
+            <h1 class="grid1">My recent works<img src="./Normal_Button/Vector.png"></h1>
```
- [x]  Please use only one h1 tag in each of your document. This is best SEO practice. Kindly change this h1 tag to any other heading tag between h2 and h5

Kamba56 reviewed (commented):
Hi @tomasesquivelgc, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed
- [x] Great job using grid and flexbox to layout your work
- [x] Great job implementing the about me and works sections.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tomasesquivelgc committed:
indentation change from 4 to 2 spaces

tomasesquivelgc committed:
fix font-family and font-weight of all elements

tomasesquivelgc committed:
fix grid elements and buttons

grid elements now 10% bigger
grid texts smaller and fit
grid technologies also smaller and fit
buttons now have both hover and active states

tomasesquivelgc committed:
fix about me section to fit text properly

text size reduce and column size increase
button smaller
li elements nest in other ul element

tomasesquivelgc committed:
background image fix

background image of headline position fix
grid1 add margin for vector image under "My works" to show

tomasesquivelgc committed:
syntax fix

tomasesquivelgc commented:
All changes done

Strangeal commented on index.html:
```diff
@@ -1,47 +1,152 @@
+  </section>
+  <section class="work-section" id="work-section">
+    <div class="work">
+      <h1 class="grid1">My recent works<img src="./Normal_Button/Vector.png"></h1> 
+      <div class="grid2">
```
As specified by the previous reviewer. Please use only one h1 tag in each of your documents. This is for best SEO practice. Kindly change this h1 tag to any other heading tag between `h2` and `h5`

Strangeal reviewed (changes requested):
Hi @tomasesquivelgc, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![Alt Text](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

 ## Highlights
Impressive! I really like how you pay attention to details in your designs. Keep it up 😊👍.


 ## Required Changes ♻️
- [x] Please add some margin to your content such that it doesn't cross the two red lines I have added on the left and right.  You can see from the second screenshot which Is the original Figma template to confirm how it has been aligned.

Your design:
<img width="328" alt="Screenshot 2023-03-25 at 13 19 53" src="https://user-images.githubusercontent.com/107192076/227740201-380b61d6-b5a8-466c-8acf-88805b0fe689.png">

Figma:
<img width="358" alt="Screenshot 2023-03-25 at 13 33 50" src="https://user-images.githubusercontent.com/107192076/227740411-94a813cf-b03c-4407-b4ae-bace740c4366.png">



_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tomasesquivelgc committed:
h1 "my recent work change to h2

grid1 (yoga girl project) centered

Tufahel reviewed (approved):
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @tomasesquivelgc  :wave: 
 
Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻

![](https://media0.giphy.com/media/d31w24psGYeekCZy/giphy.gif?cid=ecf05e47zaer8jfqmid0gfxhz2oq4gcjqy2c2aol3ldn6gvl&rid=giphy.gif&ct=g)

:heavy_check_mark: Linter checks are passing.
:heavy_check_mark: Readme is professional.


## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [![Mail](https://img.shields.io/badge/Gmail-tufahel96@gmail.com-red?logo=Gmail&logoColor=red&labelColor=black)](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
